extensions/SyntaxHighlight_GeSHi/SyntaxHighlight_GeSHi.class.php

```diff
@@ -477,4 +484,19 @@ private static function tabsToSpacesLine( $line ) {
 		}
 		return $out;
 	}
+
+	/**
+	 * Wikia change
+	 * Log GeSHi syntax highlight parsing time with 10% sampling
+	 *
+	 * @see https://wikia-inc.atlassian.net/browse/SUS-1514
+	 * @param Title $title article title of JS page or page with a syntaxhighlight tag
+	 * @param GeSHi $geSHi
+	 */
+	private static function profileGeshiPerformance( Title $title, GeSHi $geSHi ) {
+		\Wikia\Logger\WikiaLogger::instance()->debugSampled( 0.1, 'SUS-1514 - GeSHi syntax highlight performance', [
+			'titleText' => $title->getPrefixedText(),
```

Comment:
Page URL will be reported as a part of generic logger context. Do we need to log `titleText` here?